ui/src/app/base/docsUrls.ts

```diff
@@ -0,0 +1,26 @@
+const docsUrls = {
+  customisingDeployedMachines:
+    "https://maas.io/docs/how-to-customise-machines#heading--about-customising-deployed-machines",
+  dhcp: "https://maas.io/docs/dhcp",
+  sshKeys: "https://maas.io/docs/user-accounts#heading--ssh-keys",
+  rackController: "https://maas.io/docs/rack-controller",
+  networkDiscovery: "https://maas.io/docs/network-discovery",
+  configurationJourney: "https://maas.io/docs/configuration-journey",
+  cloudInit:
+    "https://maas.io/docs/custom-node-setup-preseed#heading--cloud-init",
+  kvmIntroduction: "https://maas.io/docs/kvm-introduction",
+  addNodesViaChassis:
+    "https://maas.io/docs/add-machines#heading--add-nodes-via-a-chassis",
+  addMachines: "https://maas.io/docs/add-machines",
+  images: "https://maas.io/docs/images",
+  ipmi: "https://maas.io/docs/snap/2.9/ui/power-management#heading--ipmi",
+  ipRanges: "https://maas.io/docs/ip-ranges",
+  tagsXpathExpressions:
+    "https://maas.io/docs/how-to-work-with-tags#heading--xpath-expressions",
+  tagsKernelOptions:
+    "https://maas.io/docs/how-to-work-with-tags#heading--kernel-options",
+  tagsAutomatic:
+    "https://maas.io/docs/how-to-work-with-tags#heading--automatic-tags",
+};
```

Comment:
Maybe we should be adding `as const` to these so we can't accidentally overwrite any values.

Reply:
Yeah, sure! Didn't even think anyone ever would want to do it but you're absolutely right. `const` properties should be enforced as read only by default 😒